lib/routes/bupt/scss.ts

```diff
@@ -0,0 +1,119 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import timezone from '@/utils/timezone';
+import { parseDate } from '@/utils/parse-date';
+import type { Context } from 'hono';
+
+export const route: Route = {
+    path: '/scss/:type',
+    categories: ['university'],
+    example: '/bupt/scss/xwdt',
+    parameters: {
+        type: {
+            type: 'string',
+            optional: false,
+            description: '信息类型，可选值：新闻动态，通知公告',
+        },
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['scss.bupt.edu.cn/index/xwdt.htm'],
+            target: '/scss/xwdt',
+        },
+        {
+            source: ['scss.bupt.edu.cn/index/tzgg1.htm'],
+            target: '/scss/tzgg',
+        },
+    ],
+    name: '网络空间安全学院',
+    maintainers: ['ziri2004'],
+    handler,
+    url: 'scss.bupt.edu.cn',
+};
+
+async function handler(ctx: Context) {
+    let type = ctx.req.param('type'); // 默认类型为通知公告
+    if (!type) {
+        type = 'tzgg';
+    }
```

Comment:
https://github.com/DIYgod/RSSHub/blob/c6a27229dcd56d44d18f8a6d6bbde91b948e4f96/lib/routes/bupt/scss.ts#L10 defines `type` as a compulsory parameter which means no default value is expected. User who visited without `type` will be rejected by middleware.